Rakefile

```diff
@@ -408,10 +415,17 @@ def run_awestruct(args)
 
   if ENV['site_base_path']
     base_url = ENV['site_base_path']
-    base_url = "#{base_url}/#{ENV['site_path_suffix']}" if ENV['site_path_suffix']
+
+    if ENV['site_path_suffix'].to_s != ''
```

Comment:
The idea here is to protect against nils and empty strings. 

```ruby
ENV['NOT_THERE'].chop
```

Will give an ```NoMethodError: undefined method `chop' for nil:NilClass``` if that environment variable is nil.

To make sure it does't blow up we convert to string as, 
```ruby
nil.to_s
```
returns ```""```

I do think using ```empty?``` makes the intent more clear, so I'll make that change now thanks.